### ISSUE TITLE
Faust 2 TouchDesigner?

### ISSUE THREAD

**Issue by JeremyKeenan** [grame-cncm/faust#395]
I'm wondering if Faust 2 TouchDesigner might be added at some point.

As of now there is only a very small range of native audio objects in TouchDesigner, but it has the ability to load C++ plugins:

https://docs.derivative.ca/CPlusPlus_CHOP

Faust to the rescue?

**Comment by sletz** [comment]
I don't see a clear reference to any audio SDK to start with ?

**Comment by JeremyKeenan** [comment]
No, no specific audio SDK. As I understand it, the Touch Designer CHOP sends a stream of data at a specified sample rate.

I will ask about this on the Touch Designer forum, perhaps someone can come up with an audio SDK, or perhaps just a workflow for adapting one of the preexisting Faust formats to work in TouchDesigner...
